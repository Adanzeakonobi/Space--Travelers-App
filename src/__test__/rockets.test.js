import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import RocketsPage from '../pages/RocketsPage';

const rocketsArr = [
  {
    desc: 'Some random description',
    key: '1',
    id: '1',
    name: 'Space Traveler 1',
    img: 'https://whatever.com',
    reserv: false,
  },
];

describe('Test the Rockets Component renders correctly', () => {
  it('rockets section renders', () => {
    const { container } = render(
      <Provider store={store}>
        <RocketsPage key={rocketsArr[0].id} rocket={rocketsArr[0]} />
      </Provider>
    );

    expect(container).toMatchSnapshot();
  });
});
