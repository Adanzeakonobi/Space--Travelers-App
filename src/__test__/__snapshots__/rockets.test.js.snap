// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Rockets Component renders correctly rockets section renders 1`] = `<div />`;
